plantcv/plantcv/hyperspectral/read_data.py

```diff
@@ -164,15 +166,20 @@ def read_data(filename):
     max_pixel = float(np.amax(array_data))
     min_pixel = float(np.amin(array_data))
 
-    # Create an instance of the spectral_data class
+    try:
+        wavelength_units = header_dict["wavelength units"]
+    except:
+        wavelength_units = "nm"
```

Comment:
```suggestion
    wavelength_units = header_dict.get("wavelength units")
    if wavelength_units is None:
        wavelength units = "nm"
```

This isn't effectively different, but all lines should be tested based on your changes to the test data.